### DISCUSSION TITLE
[Enhancement] [select] el-select needs a dynamic width option

### DISCUSSION THREAD

**Discussion by JohnPAfr** [element-plus/element-plus#8519] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

At the moment the only way to change the width of the select is to pass style="width: size" as attribute to the component.

It would be a great feature because right now is an option is too long, the select just hide the part that is overflowing.

### Additional comments

shorturl.at/cjEOZ  (Element plus playground)

**Comment by em1382** [comment]
Bump! It'd be great to allow for a reference-decoupled popper width.
